src/org/labkey/test/tests/SampleTypeTest.java

```diff
@@ -163,6 +172,50 @@ public void testCreateSampleTypeNoExpression()
         sampleTypeHelper.verifyDataValues(data);
     }
 
+    @Test
+    public void testMeFilterOnSampleType()
+    {
+        USER_FOR_FILTERTEST.create(this)
+                .addPermission("Folder Administrator", getProjectName());
+        String sampleType = "meFilterSamples";
+        var domainDesigner = CreateSampleTypePage.beginAt(this, getProjectName());
+        domainDesigner.setName(sampleType)
+                .addField(new FieldDefinition("size", ColumnType.Integer))
+                .addField(new FieldDefinition("user", ColumnType.User));
+        var formatDialog = domainDesigner.getFieldsPanel().getField("user").clickConditionalFormatButton();
+        formatDialog.getOpenFormatPanel()
+                .setFirstCondition(Filter.Operator.EQUAL)
+                .setFirstValue("~me~")
+                .setFillColor("#F44E3B")    // red
+                .setBoldCheckbox(true);
+        formatDialog.clickApply();
+        domainDesigner.clickSave();
+
+        var sampleHelper = new SampleTypeHelper(this).goToSampleType(sampleType);
+
+        var insertPage = sampleHelper.getSamplesDataRegionTable().clickInsertNewRow();
+        insertPage.setField("Name", "me")
+                    .setField("size", 2)
+                    .setField("user", OptionSelect.SelectOption.textOption(getDisplayName()))
+                    .submit();
+        insertPage = sampleHelper.getSamplesDataRegionTable().clickInsertNewRow();
+        insertPage.setField("Name", "not me")
+                .setField("size", 3)
+                .setField("user", OptionSelect.SelectOption.textOption(USER_FOR_FILTERTEST.getUserDisplayName()))
+                .submit();
+
+        var meCell = Locator.tag("td").withChild(Locator.tagWithText("a", getDisplayName()))
+                .waitForElement(getDriver(), WAIT_FOR_JAVASCRIPT);
+        var notMeCell = Locator.tag("td").withChild(Locator.tagWithText("a", USER_FOR_FILTERTEST.getUserDisplayName()))
+                .findElement(getDriver());
+        assertThat("expect custom format for me filter",
+                meCell.getAttribute("style"), containsString("background-color: rgb(244, 78, 59)"));
```

Comment:
FYI, `WebElement.getCssValue` will parse the style attribute for you and handle when the style attribute isn't set, preventing an NPE (`containsString` doesn't like comparing to null values if I remember correctly).
```suggestion
        assertEquals("expect custom format for me filter", "rgb(244, 78, 59)", meCell.getCssValue("background-color"));
```

Reply:
cool!  thanks